app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,33 @@
+class OrdersController < ApplicationController
+  before_action :get_order
+  before_action :get_referer
+
+  def create
+    dish = Dish.find(params[:dish_id])
+    if @order.add(dish, dish.price)
+      flash[:notice] = "#{dish.title} was successfully added to order!"
+    else
+      flash[:alert] = 'Item not added, try again!'
```

Comment:
Is there a situation when this can actually happen? If not, please remove this code.